buildconfig/CMake/DarwinSetup.cmake

```diff
@@ -124,9 +124,9 @@ set(PLUGINS_DIR plugins)
 # Mac-specific installation setup
 # ##############################################################################
 # use homebrew OpenSSL package
-if(NOT OPENSSL_ROOT_DIR)
```

Comment:
Is `OPENSSL_ROOT_DIR` an old name from an old finder?

Reply:
So `OPENSLL_ROOT_DIR` is a variable from the finder (and I think it's still present in the module). Since CMake 3.12, the first place the finders look is at the `<PackageName>_ROOT` path. For consistency, it felt more appropriate to use this variable, rather than the one defined in the find module, but both should work.

Reply:
So it looks like this has broken the [non-conda clean builds](https://builds.mantidproject.org/job/master_clean-osx/1672/console) with
```
02:14:36 CMake Error at /usr/local/Cellar/cmake/3.17.3/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
02:14:36   Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
02:14:36   system variable OPENSSL_ROOT_DIR (missing: OPENSSL_INCLUDE_DIR)
```
I get the same error message in cmake 3.20.

Reply:
I think I've just spotted the problem here and it's annoying. The OpenSSL finder root variable is `OPENSSL_ROOT_DIR` whereas the generic find root path is `OpenSSL_ROOT` because the package names are case sensitive 🤦🏻

Reply:
I guess the reason this doesn't matter in conda is we set the system prefix and openssl is in a standard location. I'll open a quick PR.